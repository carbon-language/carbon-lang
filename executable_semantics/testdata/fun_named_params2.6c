// Part of the Carbon Language project, under the Apache License v2.0 with LLVM
// Exceptions. See /LICENSE for license information.
// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception

fn f(Int: x, .d = Int: y, Int: z, .e = Int: a) => (x + y) - (z + a);

fn main() -> Int {
  return f(.d = 4, 1, 3, .e = 2);
}
