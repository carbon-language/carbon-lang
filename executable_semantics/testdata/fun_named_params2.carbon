// Part of the Carbon Language project, under the Apache License v2.0 with LLVM
// Exceptions. See /LICENSE for license information.
// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception

package ExecutableSemanticsTest api;

fn f(x: i32, .d = y: i32, z: i32, .e = a: i32) => (x + y) - (z + a);

fn main() -> i32 {
  return 0;
}
