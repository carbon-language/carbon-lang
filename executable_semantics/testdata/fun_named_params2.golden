PROGRAM ERROR: 5: positional members must come before named members
EXIT CODE: 255
